docs/creating-components.md

```diff
@@ -211,3 +211,47 @@ import CustomButton from './CustomButton';
 ```
 
 For more details and advanced theming features and recommendations, see our [Theming Guide](./theming.md).
+
+## Customizing Sandstone skin at Run-Time
+
+Enact provides several ways to customize the appearance of components but all these methods are at design time. We've got lots of requests to support runtime customization. We've added Sandstone skin customization feature at runtime. This means you can customize the appearance of Sandstone components after the application is built.
```

Comment:
Hm.. I think this part is somewhat not convincing. How about elaborating on the merits of runtime customizations with examples?
instead of
```
We've got lots of requests to support runtime customization. We've added Sandstone skin customization feature at runtime. This means you can customize the appearance of Sandstone components after the application is built.
```
I'll also think about more.